src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -2207,6 +2211,8 @@ private void ShowNewFunctionDialogAndMakeFunction(object parameter)
                 this.ExecuteCommand(new DynamoModel.CreateCustomNodeCommand(Guid.NewGuid(),
                     args.Name, args.Category, args.Description, true));
                 this.ShowStartPage = false;
+
+                SetDefaultScaleFactor();
```

Comment:
I'm very confused by this, what is the intention here? @avidit has this been tested? @RobertGlobant20 @zeusongit  It seems like when creating a new custom node, we're setting the home workspace scaling instead of the custom node workspace scaling?

Reply:
Honestly, I don't know what is supposed to happen in this case, I imagine custom nodes workspaces don't have workspace scaling .... as they don't execute? Why is this set in this function at all?

Reply:
@mjkkirschner I've added this because I was under the idea that Custom Nodes use the Geometry Scaling but not sure is needed or not.
In the future PR that I will be creating with this fixes will be selecting the one with CustomNodeWorkspaceModel (so it will be the right WorkspaceViewModel).

Reply:
we should discuss this with @aparajit-pratap - custom node workspaces themselves don't really execute, only home workspaces execute - the custom nodes are just functions - so IMO it's a bit weird for custom nodes to care about scale factor, they should be executed with the current scale factor of the home workspace.

@aparajit-pratap what do you think? Is that how it's always worked?